test/libsolidity/smtCheckerTests/modifiers/modifier_overriding_3.sol

```diff
@@ -0,0 +1,22 @@
+pragma experimental SMTChecker;
+
+abstract contract A {
+	bool s;
+
+	function f() public view mod {
+		assert(s); // holds for B
```

Comment:
Shouldn't this fail for A?

Reply:
My understanding is that `f` cannot be called for `A`, since in `A` the modifier is abstract. So there cannot be an instance of `A`. I was thinking the analysis is doing the same reasoning. Do you think that's not the case?

Reply:
True, `mod` shouldn't be called for A. Hmm I'm wondering now what makes sense. Is the code now not analyzing A.f at all?

Reply:
It seems `A.f` is not analyzed, because there is no placeholder in the body of the modifier (no `_;`) (since the modifier has no body). And I think that is the correct way.

Reply:
Ok that makes sense.